app/data_definitions/from_cascade/Shared Fields/image.xml

```diff
@@ -1,5 +1,5 @@
 <system-data-structure>
-	<group identifier="image" label="Image" multiple="true">
+	<group identifier="imageNICKWASHERE" label="Image" multiple="true">
```

Comment:
is this an easter egg or debug artifact?

Reply:
haha! oops. Will fix shortly

Reply:
Fixed below in e03430f